x/pairing/keeper/filters/filter.go

```diff
@@ -27,13 +44,16 @@ func GetAllFilters() []Filter {
 	return filters
 }
 
-func initFilters(filters []Filter, strictestPolicy planstypes.Policy) []Filter {
+func initFilters(filters []Filter, strictestPolicy planstypes.Policy) (originalFilters []Filter) {
```

Comment:
they are not original though, also the mixed filters are there

Reply:
done. see https://github.com/lavanet/lava/commit/7e3610f4d558155f11daf3e24fa73afa58b06eea